tests/integration/test_github.py

```diff
@@ -181,6 +181,11 @@ def test_pr_comments_updates(self):
 
 
 class GenericCommands(GithubTests):
+    def test_add_user(self):
+        project = self.service.get_project(repo="clynica", namespace="Beauth")
+        data = project.add_user("lachmanfrantisek", AccessLevel.pull)
```

Comment:
Nice, that you made it. This is not an easy request-response situation. Good job!